### DISCUSSION TITLE
The alshedivat/al-folio theme is not applied to create a GitHub blog.

### DISCUSSION THREAD

**Discussion by JDabeen** [community/community#131385] [Repositories]
The guidelines are as follows:
Create a new repository using this template. For this, click on Use this template -> Create a new repository above the file list. If you plan to upload your site to <your-github-username>.github.io, note that the name of your repository ⚠️ MUST BE ⚠️ <your-github-username>.github.io or <your-github-orgname> .github.io, as stated in the GitHub pages docs.
In this new repository, go to Settings -> Actions -> General -> Workflow permissions and give Read and write permissions to GitHub Actions.
Open file _config.yml, set url to https://<your-github-username>.github.io and leave baseurl empty (do NOT delete it).
Wait until the GitHub actions finish (check your repository Actions tab). Now, in addition to the master branch, your repository has a newly built gh-pages branch.
Finally, in the repository page go to Settings -> Pages -> Build and deployment, make sure that Source is set to Deploy from a branch and set the branch to gh-pages (NOT to master).
Wait until the GitHub actions finish (check your repository Actions tab), then simply navigate to https://<your-github-username>.github.io in your browser. At this point you should see a copy of the theme's demo website.

Here, it asks you to create gh-pages in step 5. How do you create gh-pages? Didn't it happen automatically if you change the _config.yml settings?

**Comment by Akash1134** [comment]
Welcome to the GitHub Community, @JDabeen , we're happy you're here! You are more likely to get a useful response if you are posting your question(s) in the applicable category and are explicit about what your project entails--giving a few more details might help someone give you a nudge in the right direction. I've gone ahead and moved it for you. Good luck!
